controller.py

```diff
@@ -493,6 +492,15 @@ def register_identifier_as_unresolved(self, urn, identifier):
         # work gets done eventually by creating a CoverageRecord
         # representing the work that needs to be done.
         source = DataSource.lookup(self._db, DataSource.INTERNAL_PROCESSING)
+
+        # Add this Identifier to the 'unaffiliated' collection to make
+        # sure it gets covered eventually by the identifier resolution
+        # script, which only covers Identifiers that are in some
+        # collection.
+        collection, ignore = IdentifierResolutionCoverageProvider.unaffiliated_collection(
```

Comment:
If you add identifiers to the collection here, every unresolved identifier will be marked as unaffiliated, regardless of their attachment to other collections. There's a segment in `process_urn` that catalogs Identifiers in an indicated collection. What do you think of cataloguing unaffiliated Identifiers at that point only if no collection has been identified?

Reply:
A good idea.